romtools/__init__.py

```diff
@@ -125,11 +125,14 @@ def getNumVars(self) -> int:
     snapshots = HeatSnapshots(data)
 ```
 
-
-
+# License
+```plaintext
+.. include:: ../LICENSE
+```
 '''
 
-__docformat__ = "markdown"  # explicitly disable rST processing in the examples above.
+__docformat__ = "restructuredtext"
+# __docformat__ = "markdown"  # explicitly disable rST processing in the examples above.
 
```

Comment:
I had to enable rST in order to include the license file in the documentation. I could not see anything being affected by it--why were we explicitly disabling rST processing?

Reply:
becuase i followed what pdoc does in its doc, but if you say this does not change anything it is fine